pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -54,6 +54,20 @@ var (
 		metrics.ALPHA,
 		"")
 
+	podCPUUsageDesc = metrics.NewDesc("pod_cpu_usage_seconds_total",
+		"Cumulative cpu time consumed by the pod in core-seconds",
+		[]string{"pod", "namespace"},
+		nil,
+		metrics.ALPHA,
+		"")
+
+	podMemoryUsageDesc = metrics.NewDesc("pod_memory_working_set_bytes",
+		"Current working set of the pod in bytes",
+		[]string{"pod", "namespace"},
+		nil,
+		metrics.ALPHA,
+		"")
+
 	resouceScrapeResultDesc = metrics.NewDesc("scrape_error",
```

Comment:
while you on it, can you fix spelling of the variable?

Reply:
sure.

Reply:
Done.